src/app.ts

```diff
@@ -461,34 +464,6 @@ export class App {
     return dir;
   }
 
-  /**
```

Comment:
No longer consumed or needed, since we've replaced it with a simpler method write.